robotpy-wpilib/wpilib/__init__.py

```diff
@@ -1,6 +1,417 @@
 
 from . import _init_wpilib
-from ._wpilib import *
+
+# autogenerated by 'robotpy-build create-imports wpilib wpilib._wpilib'
+from ._wpilib import (
+    ADXL345_I2C,
+    ADXL345_SPI,
+    ADXL362,
+    ADXRS450_Gyro,
+    AnalogAccelerometer,
+    AnalogEncoder,
+    AnalogGyro,
+    AnalogInput,
+    AnalogOutput,
+    AnalogPotentiometer,
+    AnalogTrigger,
+    AnalogTriggerOutput,
+    AnalogTriggerType,
+    BuiltInAccelerometer,
+    CAN,
+    CANData,
+    CANStatus,
+    Color,
+    Color8Bit,
+    Compressor,
+    Counter,
+    DMC60,
+    DigitalGlitchFilter,
+    DigitalInput,
+    DigitalOutput,
+    DigitalSource,
+    DoubleSolenoid,
+    DriverStation,
+    DutyCycle,
+    DutyCycleEncoder,
+    Encoder,
+    Error,
+    ErrorBase,
+    GyroBase,
+    HAL_GetErrorMessage,
```

Comment:
wait why is this showing up here

Reply:
`... import *` works just fine but pylint doesn't detect anything and puts red squigglies everywhere

Reply:
no I mean why are bits of the HAL leaking into our wpilibc bindings